fdbt-dev/data/matchingData/schoolServicePointToPoint.json

```diff
@@ -0,0 +1,554 @@
+      },
+      {
+        "atcoCode": "2500B0359"
+      },
+      {
+        "atcoCode": "2500B0360"
+      },
+      {
+        "atcoCode": "2500B0266"
+      },
+      {
+        "atcoCode": "2500B0267"
+      },
+      {
+        "atcoCode": "2500B0268"
+      },
+      {
+        "atcoCode": "2500B0269"
+      },
+      {
+        "atcoCode": "2500B0534"
+      },
+      {
+        "atcoCode": "2500B0535"
+      },
+      {
+        "atcoCode": "2500IMG660"
+      },
+      {
+        "atcoCode": "2500IMG661"
+      },
+      {
+        "atcoCode": "2500IMG663"
+      },
+      {
+        "atcoCode": "2500B0449"
+      }
+    ],
+    "inboundUnassignedStops": [
+      {
+        "atcoCode": "2500B0452"
+      },
+      {
+        "atcoCode": "2500B0453"
+      },
+      {
+        "atcoCode": "2500B0454"
+      },
+      {
+        "atcoCode": "2500B0455"
+      },
+      {
+        "atcoCode": "2500LAA15912"
+      },
+      {
+        "atcoCode": "2500B0271"
+      },
+      {
+        "atcoCode": "2500B0272"
+      },
+      {
+        "atcoCode": "250012414"
+      },
+      {
+        "atcoCode": "2500B0274"
+      },
+      {
+        "atcoCode": "2500B0361"
+      },
+      {
+        "atcoCode": "2500B0362"
+      },
+      {
+        "atcoCode": "2500607"
+      },
+      {
+        "atcoCode": "2500B0364"
+      },
+      {
+        "atcoCode": "2500B0365"
+      },
+      {
+        "atcoCode": "2500B0366"
+      },
+      {
+        "atcoCode": "2500B0367"
+      },
+      {
+        "atcoCode": "2500B0368"
+      },
+      {
+        "atcoCode": "2500B0369"
+      },
+      {
+        "atcoCode": "2500B0370"
+      },
+      {
+        "atcoCode": "2500B0371"
+      },
+      {
+        "atcoCode": "2500BTA01069"
+      },
+      {
+        "atcoCode": "2500686"
+      },
+      {
+        "atcoCode": "2500B0859"
+      },
+      {
+        "atcoCode": "2500B0389"
+      },
+      {
+        "atcoCode": "2500B0390"
+      },
+      {
+        "atcoCode": "2500B0391"
+      },
+      {
+        "atcoCode": "2500IMG396"
+      },
+      {
+        "atcoCode": "2500B0393"
+      },
+      {
+        "atcoCode": "2500B0394"
+      },
+      {
+        "atcoCode": "2500B0395"
+      },
+      {
+        "atcoCode": "2500B0396"
+      },
+      {
+        "atcoCode": "2500B0397"
+      },
+      {
+        "atcoCode": "2500B0398"
+      },
+      {
+        "atcoCode": "2500IMG388"
+      },
+      {
+        "atcoCode": "2500IMG389"
+      },
+      {
+        "atcoCode": "2500LAA16018"
+      },
+      {
+        "atcoCode": "2500B0400"
+      },
+      {
+        "atcoCode": "2500B0401"
+      },
+      {
+        "atcoCode": "2500B0402"
+      },
+      {
+        "atcoCode": "2500B0403"
+      },
+      {
+        "atcoCode": "2590B0404"
+      },
+      {
+        "atcoCode": "2590B0405"
+      },
+      {
+        "atcoCode": "2590B0406"
+      },
+      {
+        "atcoCode": "2590B0407"
+      },
+      {
```

Comment:
Can we remove most of these atcos? It just makes the json very long for no benefit.

Reply:
reduced it to 5